### DISCUSSION TITLE
It used to work for my goguardian but now it does not but it still works for my friend

### DISCUSSION THREAD

**Discussion by ItsBubby** [3kh0/ext-remover#590] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I used to work but my computer crashed with it on and now it does not work for GoGuardian and I do not think it works for any admin-added thing now. It could be that my Chromebook crashed but it also updated my friend's Chromebook also updated and it still works for him so pls help.

**Comment by S-PScripts** [comment, marked as answer]
jjkdsakdjg
